src/main/java/commission/entity/Type.java

```diff
@@ -0,0 +1,7 @@
+package commission.entity;
+
+public enum Type {
```

Comment:
This a bit of a generic name, yes you can infer it from the package name but I prefer to be explicit and rather name it something like `CommissionType`

Reply:
Yes I was not happy with that name.  Type is way to common.